core/config.go

```diff
@@ -121,7 +124,8 @@ func ParseConfig(conf *Config) {
 	conf.LogFlag = log.Ldate | log.Ltime | log.Lshortfile
 	conf.MainRegion = region
 	conf.SleepMultiplier = 1
-	conf.LambdaManageASG = os.Getenv("LAMBDA_MANAGE_ASG")
+	conf.SQSQueueURL = os.Getenv("SQS_QUEUE_URL")
```

Comment:
As far as I know the command like flag library we use shouldn't need this, it's automatically taking env var `VAR_NAME` if we have a flag `-var-name`, and I see we have it defined below.